sys/net/gnrc/sock/ip/gnrc_sock_ip.c

```diff
@@ -36,7 +36,7 @@ int sock_ip_create(sock_ip_t *sock, const sock_ip_ep_t *local,
         (local->netif != remote->netif)) {
         return -EINVAL;
     }
-    memset(&sock->local, 0, sizeof(sock_ip_t));
+    memset(&sock->local, 0, sizeof(sock_ip_ep_t));
```

Comment:
Please consider my suggestion about using a variable instead of a type. This bug wouldn't exist if we did it that way.